src/vue/common/Drawer.vue

```diff
@@ -44,8 +54,24 @@ export default {
     isShown: { type: Boolean, default: true },
     closeByClickOutside: { type: Boolean, default: true },
   },
+  data: () => ({
+    isHidden: true,
+    isDeleted: true,
+  }),
+  watch: {
+    'isShown': function () {
+      if (this.isShown) {
```

Comment:
if (value) {

Reply:
Because if `isShown` name will be changed, you don't need to change it in this condition.